core/audits/bf-cache.js

```diff
@@ -70,6 +72,19 @@ class BFCache extends Audit {
    * @return {Promise<LH.Audit.Product>}
    */
   static async audit(artifacts) {
+    // Old headless mode does not obfuscate the specific Chrome version in the UA string
+    // Old Headless Example: HeadlessChrome/120.0.6099.1
+    // New Headless Example: HeadlessChrome/120.0.0.0
+    const isOldHeadless =
+      /HeadlessChrome\/[0-9]+\.[0-9]+.[0-9]{2,}.[0-9]+/.test(artifacts.HostUserAgent);
```

Comment:
This condition works but it's a bit sketchy. Might be better to just use the `product` string from `Browser.getVersion` which differentiates between the old&new headless better.

Reply:
Update: This condition is not stable and main change in the future.